node/config/def.go

```diff
@@ -65,8 +65,9 @@ func DefaultBoost() *Boost {
 		},
 
 		Graphql: GraphqlConfig{
-			ListenAddress: "127.0.0.1",
-			Port:          8080,
+			ListenAddress:      "127.0.0.1",
+			Port:               8080,
+			PendingAlertEpochs: 10,
```

Comment:
I'd suggest
- we move the config somewhere else (it's not directly to do with graphql)
  - maybe at the top level? not sure where exactly would make sense
- we change the name to `MpoolAlertEpochs`
- we start with a value of 30 (I believe lotus tries to estimate gas fees such that messages will get sent within 20 epochs)